utopia-remix/app/routes/v1.github.save.$projectId.tsx

```diff
@@ -10,6 +11,6 @@ export async function loader(args: LoaderFunctionArgs) {
 
 export async function action(args: ActionFunctionArgs) {
   return handle(args, {
-    POST: proxy,
+    POST: { handler: proxy, validator: ALLOW },
```

Comment:
this should also validate the project access

Reply:
fixed